cartographer_docker/README.md

```diff
@@ -0,0 +1,82 @@
+# Description
+
+This is a docker image with installed code from the [Roboy Google Cartographer repository](https://github.com/Roboy/cartographer_ros).
+Once a container of this image is created, it will run a roscore exposed to the 11311 port.
+
+## Setup
+
+To build the image run 
+```
+sudo docker build -t roboy_cartographer . --no-cache
+```
+
+To create a docker container run 
+```
+sudo docker run -it -d -p 11111:11311 --network=host --name roboy_cartographer roboy_cartographer:latest bash
```

Comment:
NIT: if docker is in the same network and there's only one ROS core running, we don't need to map ports anymore, right?

Reply:
yep, that is one of the advantages of this solution.